nox/sessions.py

```diff
@@ -177,6 +178,19 @@ def __init__(self, name, signature, func, global_config):
 
     def _create_config(self):
         self.config = SessionConfig(posargs=self.global_config.posargs)
+
+        # By default, nox should quietly change to the directory where
+        # the nox.py file is located; however, do not be overly stringent
+        # about this. If we cannot easily inspect that location, just give up
+        # and skip that command.
+        try:
+            cwd = os.path.dirname(inspect.getsourcefile(self.func))
```

Comment:
I did not notice that. :-)

This would be a slight behavior difference, as if the noxfile imported a function from another directory, the noxfile's directory would be used, not the imported function's directory (which is the behavior I would want). I _think_ I do not care about that distinction, at least not yet, but it is worth asking.